lib/errors/sql-errors.js

```diff
@@ -1,10 +1,10 @@
-var util = require('util');
+var util = require("util");
 
 /**
  * Specific error for this library so we can tell it apart from other errors.
  * @class
- * @param {String} message
- * @param {String} [key]
+ * @param {String} message   error message
```

Comment:
Is there a need for `error message` its the `message` for the error. I see this the same as

    element.id               // <- better, in my opinion
    element.elementId